src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -54,3 +61,14 @@ const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | Man
         return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
   }
 };
+
+const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
+  const pscData = await getCompanyPsc(req, appData[EntityNumberKey] as string);
+  const session = req.session as Session;
+
+  // setting key to [] for all scenarios right now
+  // UAR-337 to update
+  if (pscData || pscData === undefined ) {
```

Comment:
effectively "if (true)" so remove if?

Reply:
yeah I didn't like this code tbh, it was just put in temporarily for Olayode but he can add in his ticket. I'll remove it

Reply:
fixed